src/components/landing/SideBySideHero/SideBySideHero.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { Headline } from 'components';
+import CatalogScreenshot from '../../../../content/assets/homepage/home-product-screenshot.png';
+import { LOGOS } from '../CustomerLogoCloud';
+import has from 'lodash/has';
+import { Button } from 'components';
+
+const SideBySideHero = () => {
+  const whiteLogos = [...LOGOS].filter(({ src }) => has(src, 'white')).slice(0, 5);
+
+  return (
+    <section className="landing-hero-background mx-auto max-w-7xl xl:rounded-lg lg:flex items-center">
+      <div className="lg:w-1/2 p-4 lg:px-10 lg:py-16">
+        <Headline size="small" className="text-white leading-snug">
+          <nobr>Backstage single-tenant SaaS:</nobr> <br /> adopt without the overhead
```

Comment:
I know I was the one who brought this up, but I think mentioning "single-tenant" here is too much. It's not *that* much of a selling point and it's a technical term which feels out of place in a headline.